mapie/estimators.py

```diff
@@ -36,54 +37,55 @@ class MapieRegressor(BaseEstimator, RegressorMixin):  # type: ignore
     method: str, optional
         Method to choose for prediction interval estimates.
         Choose among:
-        - "naive"
-        - "jackknife"
-        - "jackknife_plus"
-        - "jackknife_minmax"
-        - "cv"
-        - "cv_plus"
-        - "cv_minmax"
 
-        By default "cv_plus".
+        - "naive", based on training set residuals,
+        - "base", based on cross-validation sets residuals,
+        - "plus", based on cross-validation sets residuals and testing set predictions,
+        - "minmax", based on cross-validation sets residuals and testing set predictions
+          (min/max among cross-validation clones).
 
-    n_splits: int, optional
-        Number of splits for cross-validation, by default 5.
+        By default "plus".
 
-    shuffle: bool, optional
-        Whether to shuffle the data before splitting into batches, by default True.
+    cv: Optional[Union[int, BaseCrossValidator]]
+        The cross-validation strategy for computing residuals. It directly drives the
+        distinction between jackknife and cv variants. Choose among:
+
+        - `sklearn.model_selection.LeaveOneOut()`, jacknife variants are used,
```

Comment:
Je reprendrais l'ordre donné dans `CalibratedClassifierCV` (ainsi que `GridSearchCV`, etc): 
- None, to use the default 5-fold cross-validation
- integer, to specify the number of folds
- CV splitter (`sklearn.model_selection.LeaveOneOut()` if jackknife variants are used, `sklearn.model_selection.KFold()` if cross-validation variants are used)

Reply:
Done